sdk/tests/deqp/functional/gles3/es3fShaderOperatorTests.js

```diff
@@ -2663,6 +2664,10 @@ es3fShaderOperatorTests.ShaderOperatorTests.prototype.init = function() {
         all, es3fShaderOperatorTests.unaryGenTypeFuncs(Math.trunc)));
     comm.push(op("round", "round", GT, [v(GT, 2.5, 2.5)], f(0.2), f(0.7),
         all, es3fShaderOperatorTests.unaryGenTypeFuncs(roundToEven)));
+
+    comm = new es3fShaderOperatorTests.BuiltinFuncGroup("common_functions", "Common function tests.");
```

Comment:
I'm not really familiar with this stuff, but should these be numbered? `_01`, etc

Reply:
By convention, higher up in this file, they aren't numbered. Doing so changes the dEQP's notion of the test name, and global suppressions wouldn't work any more

Reply:
Ah, I see. LGTM